app/Module/GoogleMapsModule.php

```diff
@@ -4692,8 +4704,10 @@ function initialize() {
 			//--------------------------------------------------------------------------------------
 			var street = new google.maps.ImageMapType({
 				getTileUrl: function(coord, zoom) {
-					var X = coord.x % (1 << zoom);  // wrap
-					return 'https://cbk0.google.com/cbk?output=overlay&zoom=' + zoom + '&x=' + X + '&y=' + coord.y + '&cb_client=api';
+					//var X = coord.x % (1 << zoom);  // wrap
+					//return 'https://cbk0.google.com/cbk?output=overlay&zoom=' + zoom + '&x=' + X + '&y=' + coord.y + '&cb_client=api';
+					//return "http://tile.openstreetmap.org/" + zoom + "/" + X + "/" + coord.y + ".png";
+					return '';
```

Comment:
This seems to be creating an empty overlay `street`, which we add to the map a few lines further down.

Removing it doesn't seem to break anything.  Do you know what it is trying to do?  Perhaps this is (was) the code that created the "blue lines" when drag the "pegman" over a road?